cmd/prometheus/main_windows_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +build windows
+
+package main
+
+import (
+	"os"
+)
+
+func stopProcess(p *os.Process) {
+	p.Kill()
```

Comment:
Why do we not stop all OSes the same way?

Reply:
It is preferrable to stop with Interrupt as it gives Prometheus the ability to cleanly shutdown.

However Interrupt is not supported in Windows.

Reply:
KILL is supported on Unix.

Reply:
does it trigger a graceful shutdown of TSDB?

Reply:
No, but you have that exact problem on Windows anyway.